### PR TITLE
[CI] Build wasmedge on Nix

### PR DESCRIPTION
**Pull request by dannypsnl**
Build wasmedge on Nix

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: .github/workflows/build_for_nix.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      - "CMakeLists.txt"
+      - "cmake/**"
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/cache@v2
```
Comment (hydai) [inline .github/workflows/build_for_nix.yml:42]: We should use v3

**Review by hydai — CHANGES_REQUESTED** [review]
Could we run unit tests together?

**Comment by dannypsnl** [comment]
@hydai That would have to modify `flake.nix`, should we do that in this PR?

**Comment by hydai** [comment]
We can have another one to do this.

**Comment by dannypsnl** [comment]
@hydai I create a tracker for that #2679

**Comment by hydai** [comment]
The CI failed :-(

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**